src/docs/Components/Form/PhoneNumberInput/accessibility.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Accessibility'
+order: 3
+---
+
+## Accessibility and semantic
+<HintItem>
+The button element (mc-phone-number__button) must always have an id, a aria-haspopup and aria-labelledby attribute.
+</HintItem>
+<HintItem>
+The list element (mc-phone-number__list) must always have a role list or listbox
+</HintItem>
+<HintItem>
+Each list element must have the role `option`
+</HintItem>
+<HintItem>
+Add the `aria-selected="true"` attribute to the selected list element.
```

Comment:
Add the <strong>aria-selected="true"</strong> attribute to the selected list element.